modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/scope/logic/HttpMethodScopeLogic.java

```diff
@@ -49,58 +40,22 @@ public boolean check(
 		Class<?> resourceClass, Method resourceMethod,
 		ScopeChecker scopeChecker) {
 
-		return check(
+		return HttpMethodScopeLogicUtil.check(
 			propertyAccessorFunction, _getHttpMethod(resourceMethod),
 			scopeChecker);
 	}
 
-	public boolean check(
-		Function<String, Object> propertyAccessorFunction, String requestMethod,
-		ScopeChecker scopeChecker) {
-
-		try {
-			String applicationName = GetterUtil.getString(
-				propertyAccessorFunction.apply("osgi.jaxrs.name"));
-
-			Object ignoreMissingScopesObject = propertyAccessorFunction.apply(
-				"ignore.missing.scopes");
-
-			Set<String> ignoreMissingScopes = _ignoreMissingScopes;
-
-			if (ignoreMissingScopesObject != null) {
-				ignoreMissingScopes = new HashSet<>(
-					StringPlus.asList(ignoreMissingScopesObject));
-			}
-
-			ScopeFinder scopeFinder = _bundleContext.getService(
-				_getServiceReference(applicationName, ScopeFinder.class));
-
-			Collection<String> scopes = scopeFinder.findScopes();
-
-			if ((!scopes.contains(requestMethod) &&
-				 ignoreMissingScopes.contains(requestMethod)) ||
-				scopeChecker.checkScope(requestMethod)) {
-
-				return true;
-			}
-		}
-		catch (Exception exception) {
-			if (_log.isWarnEnabled()) {
-				_log.warn(exception);
-			}
-		}
-
-		return false;
-	}
-
 	@Activate
 	protected void activate(
 		BundleContext bundleContext, Map<String, Object> properties) {
 
 		_bundleContext = bundleContext;
 
-		_ignoreMissingScopes = new HashSet<>(
-			StringPlus.asList(properties.get("ignore.missing.scopes")));
+		_ignoreMissingScopes = new HashSet<>();
```

Comment:
Hey @Valmir-unicap, you do not need this entire method and the variables within it. Please remove.